functional-tests/resources/file.yaml

```diff
@@ -0,0 +1,13 @@
+first_name: John
+last_name: Doe
+age: 45
+phone_numbers:
+  - type: work
+    number: '1234567890'
```

Comment:
that's not a number, that's a string

Reply:
Phone numbers are sometimes not just numbers, see 17) and the following here https://github.com/googlei18n/libphonenumber/blob/master/FALSEHOODS.md